data/json/npcs/TALK_COMMON_ALLY.json

```
@@ -46,7 +46,71 @@
     "responses": [
       { "text": "I want to give you some commands for combat.", "topic": "TALK_COMBAT_COMMANDS" },
       { "text": "I want to set some miscellaneous rules.", "topic": "TALK_MISC_RULES" },
+      {
+        "text": "I'd like to know a bit more about your abilities.",
+        "topic": "TALK_FRIEND",
+        "effect": "reveal_stats"
+      },
+      { "text": "There's something I want you to do.", "topic": "TALK_ALLY_ORDERS" },
+      { "text": "I just wanted to talk for a bit.", "topic": "TALK_ALLY_SOCIAL" },
+      { "text": "Can you help me understand something?", "topic": "TALK_ALLY_HELP" },
```

```suggestion
      { "text": "Can you help me understand something? (NPC TUTORIAL)", "topic": "TALK_ALLY_HELP" },
```

Please make it obvious that there is an NPC tutorial in the top level menu, for what little good it does.

---

It might do more good if there aren't 20 options to read through

---

It still needs to be as obvious as possible.

---

I agree just haven't been near a computer to fix it for a bit yet